packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.scss

```diff
@@ -53,7 +53,7 @@ $isPaddedMargin: 24px;
 
   &.cellIsCheck {
     position: relative;
-    padding: 6px 8px;
+    padding: 0;
```

Comment:
Shouldn't this have a height somewhere?

Reply:
Can you set focus to the select all button and see the focus rect?

Reply:
Height is set in .cell (32px).  Should the DetailsHeader be 40px then?

Reply:
no probably not. But it should be the same height as the other cells in the header I believe.